libimage/image_test.go

```diff
@@ -0,0 +1,151 @@
+package libimage
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestImageFunctions(t *testing.T) {
+	busyboxLatest := "docker.io/library/busybox:latest"
+	busyboxDigest := "docker.io/library/busybox@"
```

Comment:
I would encourage you to use `quay.io/libpod/busybox` instead, to avoid rate limiting